torch/_inductor/codegen/wrapper.py

```diff
@@ -513,17 +516,35 @@ def add_benchmark_harness(self, output):
                 )
             output.writeline("else:")
             with output.indent():
-                output.writeline("with maybe_profile(args.profile) as p:")
+                output.writeline("times = 10")
```

Comment:
Let's move this stuff into a helper function `import`ed from the generated code.  It is better not to repeat a ton of stuff in every generated file.